casper/contracts/simple_casper.v.py

```diff
@@ -448,7 +445,15 @@ def highest_finalized_epoch(min_total_deposits: wei_value) -> int128:
 
 @public
 @constant
-def havent_voted_and_source_target_right(
+def in_dynasty(validator_index:int128, _dynasty:int128) -> bool:
```

Comment:
This should be @private unless we have a specific case that clients would need it public.